cmd/node/config/config.toml

```diff
@@ -555,7 +555,7 @@
 
 [EpochStartConfig]
     MinRoundsBetweenEpochs = 20
-    RoundsPerEpoch         = 200
+    RoundsPerEpoch         = 100
```

Comment:
maybe change this back before merge to dev.

Reply:
will do after testing

Reply:
done